dashboard/config/foorm/forms/surveys/pd/csf_deep_dive_post.0.json

```diff
@@ -0,0 +1,604 @@
+{
+  "title": "CS Fundamentals Deep Dive Post Survey",
+  "pages": [
+    {
+      "name": "overview",
+      "elements": [
+        {
+          "type": "html",
+          "name": "overview_text",
+          "hideNumber": true,
+          "html": "<p>Thank you for spending the day with us at the CS Fundamentals Deep Dive workshop. Your participation in the workshop and responses to this post workshop survey will help Code.org and your facilitator(s) continue to reflect and adjust.</p>\n\n<p>The purpose of this survey is to:</p>\n<ul>\n    <li>Help us understand if the workshop supported your needs and development.</li>\n    <li>Give us ideas about how we might improve the workshop for the future.</li>\n</ul>\n<p>\nYour <strong>honesty and candor</strong> are appreciated. This survey should take about 10 minutes to complete.\n</p>"
+        },
+        {
+          "type": "html",
+          "name": "understand_plans_heading",
+          "hideNumber": true,
+          "html": "<h2>Understanding your plans for CS Fundamentals</h2>"
```

Comment:
And should we consider adding a type for headings rather than relying on styling inside the html?

Reply:
Maybe we should start a ticket for wishlist items and figure out which are most valuable?